bindep.txt

```diff
@@ -0,0 +1,15 @@
+texlive [platform:rpm]
+texlive-fncychap [platform:rpm]
+texlive-titlesec [platform:rpm]
+texlive-tabulary [platform:rpm]
+texlive-framed [platform:rpm]
+texlive-wrapfig [platform:rpm]
+texlive-upquote [platform:rpm]
+texlive-capt-of [platform:rpm]
+texlive-needspace [platform:rpm]
+texlive-polyglossia [platform:rpm]
+texlive-luatex85 [platform:rpm]
+texlive-anyfontsize [platform:rpm]
+texlive-ctablestack [platform:rpm]
+texlive-gnu-freefont [platform:rpm]
+latexmk [platform:rpm]
```

Comment:
Is python needed here?

Reply:
It would be unnecessary. `bindep` relies on Python, so without that we can't use bindep :smile:

Reply:
I see

Reply:
I noticed graphviz and imagemagick are missing.
I think this Dockerfile is helpful for listing up dependencies.
https://github.com/sphinx-doc/docker-ci/blob/master/Dockerfile

Reply:
Good point. I'll add those.